docs/platforms/javascript/common/configuration/integrations/onuncaughtexception.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: OnUncaughtException
+description: "Registers handlers to capture global uncaught exceptions. (default)"
+supported:
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.gcp-functions
+  - javascript.koa
+  - javascript.electron
+---
+
+_Import name: `Sentry.onUncaughtExceptionIntegration`_
+
+This integration is enabled by default. If you'd like to modify your default integrations, read [this](./../#modifying-default-integrations).
+
+The `onUncaughtExceptionIntegration` registers handlers to capture global uncaught exceptions that would cause the process to exit.
+
+This integration _does not_ prevent the process from exiting! If yo want to prevent the process from exiting, you should register your own uncaught exception handler and configure `exitEvenIfOtherHandlersAreRegistered: false` in the integration options.
```

Comment:
```suggestion
This integration _does not_ prevent the process from exiting! If you want to prevent the process from exiting, you should register your own uncaught exception handler and configure `exitEvenIfOtherHandlersAreRegistered: false` in the integration options.
```